src/routes/SignUp.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, Navigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import * as Yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { selectAuth } from '../redux/store';
+import { registerUser } from '../redux/auth/authActions';
+import useSession from '../hooks/useSession';
+import Input from '../components/Input';
+import Button from '../components/Button';
+
+const SignUp = () => {
```

Comment:
- [ ] Every time I sign up or login, I get this error:

![image](https://github.com/Atril33/DocConnect-Frontend/assets/57421823/f14a10f7-4540-4237-b1d4-9004f96e3f57)

I can get into the app, but the error always shows up "Couldn't find an active session"

Kindly, fix this error so your app behaves correctly. 👍